tests/tasks/generators/clusterloader/load.yaml

```diff
@@ -101,6 +109,16 @@ spec:
                 containerPort: 8005
         remoteWrite:
           - url: http://localhost:8005/workspaces/$(params.amp-workspace-id)/api/v1/remote_write
+            queueConfig:
+              capacity: 2500
```

Comment:
why do we need this ? Defaults are not enough for us ?

Reply:
can we document why we need these values ?

Reply:
i read an aws prometheus blog and they suggested these values. So better to keep this.

Reply:
https://aws.amazon.com/blogs/mt/getting-started-amazon-managed-service-for-prometheus/